pkg/build_model.sh

```
@@ -0,0 +1,3 @@
+# model pkg command, run when model changes, or need a local copy for final pkg
```

Shellcheck (0.11.0)

[error] Tips depend on target shell and yours is unknown. Add a shebang or a 'shell' directive.

(SC2148)